stdlib/3/asyncio/base_events.pyi

```diff
@@ -76,21 +78,21 @@ class BaseEventLoop(AbstractEventLoop):
                       family: int = ..., flags: int = ...,
                       sock: None = ..., backlog: int = ..., ssl: _SSLContext = ...,
                       reuse_address: Optional[bool] = ...,
-                      reuse_port: Optional[bool] = ...) -> Generator[Any, None, AbstractServer]: ...
+                      reuse_port: Optional[bool] = ...) -> Generator[Any, None, Server]: ...
     @overload
     @coroutine
     def create_server(self, protocol_factory: _ProtocolFactory, host: None = ..., port: None = ..., *,
                       family: int = ..., flags: int = ...,
                       sock: socket, backlog: int = ..., ssl: _SSLContext = ...,
                       reuse_address: Optional[bool] = ...,
-                      reuse_port: Optional[bool] = ...) -> Generator[Any, None, AbstractServer]: ...
+                      reuse_port: Optional[bool] = ...) -> Generator[Any, None, Server]: ...
     @coroutine
     def create_unix_connection(self, protocol_factory: _ProtocolFactory, path: str, *,
                                ssl: _SSLContext = ..., sock: Optional[socket] = ...,
                                server_hostname: str = ...) -> Generator[Any, None, _TransProtPair]: ...
     @coroutine
     def create_unix_server(self, protocol_factory: _ProtocolFactory, path: str, *,
-                           sock: Optional[socket] = ..., backlog: int = ..., ssl: _SSLContext = ...) -> Generator[Any, None, AbstractServer]: ...
+                           sock: Optional[socket] = ..., backlog: int = ..., ssl: _SSLContext = ...) -> Generator[Any, None, Server]: ...
```

Comment:
`create_unix_server()` is actually not implemented by `BaseEventLoop`. Just tested this and it throws a `NotImplementedError` when called. So they should just be removed from here.

Reply:
Removed. I believe the same is true of `create_unix_connection` and it should probably be removed as well. I haven't done that yet (as it's a bit of a separate issue than what this PR is intending) but can easily do so if you'd like.

Reply:
@scottbelden I think removing `create_unix_connection` as well would be a good idea. But that leaves us with the problem that mypy complains that `create_unix_*` is missing, although it's abstract in the base class and `BaseEventLoop` is not abstract. I see several possible solutions:

1. Restore `create_unix_server` as a copy of `AbstractEventLoop`s signature and leave `create_unix_connection` alone. I don't like this, because it doesn't reflect reality.
2. Add `ABCMeta` as metaclass to `BaseEventLoop`. That even worse, since `BaseEventLoop` is clearly not meant to be abstract and this can cause problems down the line.
3. Just add `# type: ignore` to line 22 to force this error to go away. Works, but it's a bit rough, and means that classes deriving from `BaseEventLoop` need to do the same.
4. Remove `@abstractmethod` from `AbstractEventLoop.create_unix_server` and `create_unix_connection` (and instead add a comment with an explanation). I prefer this solution as these methods are clearly not abstract in the sense that they *need* to be implemented by any concrete sub-class (which is what `@abstractmethod` implies).

I'd like to hear @JelleZijlstra's thoughts about this, though.

Reply:
So if I understand correctly, the problem is:

- The base abstract class, `AbstractEventLoop`, has an abstract method `create_unix_server`
- `BaseEventLoop` is meant to be concrete but does not actually implement `create_unix_server`.

But the second part isn't true: BaseEventLoop has a number of unimplemented methods, and the actual runtime type of an event loop is `asyncio.unix_events._UnixSelectorEventLoop` for me (and presumably some other class on Windows). So it should be fine to add ABCMeta to BaseEventLoop.

Reply:
Ah, if `BaseEventLoop` is not actually meant to be instantiated, I agree.

Reply:
If I understand correctly, `class BaseEventLoop(AbstractEventLoop)` should be changed to `class BaseEventLoop(AbstractEventLoop, metaclass=ABCMeta)`. Is this correct? Should the `create_unix_*` methods still be removed from `BaseEventLoop`?

Reply:
@scottbelden Yes to both questions. Hopefully that will resolve the build failure.

Reply:
(Also, it's easier for us if you don't force-push. We will squash merge anyway.)